client/my-sites/email/email-management/gsuite-user-item/index.jsx

```diff
@@ -20,40 +20,67 @@ import './style.scss';
 function GSuiteUserItem( props ) {
 	const translate = useTranslate();
 	const [ dialogVisible, setDialogVisible ] = useState( false );
+
 	const onFixClickHandler = ( e ) => {
 		e.preventDefault();
 		setDialogVisible( true );
 	};
+
 	const onCloseClickHandler = () => {
 		setDialogVisible( false );
 	};
 
+	const getMailboxLink = () => {
+		const { email } = props.user;
+
+		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https://mail.google.com/`;
+	};
+
 	const getLoginLink = () => {
 		const { email, domain } = props.user;
+
 		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https://admin.google.com/a/${ domain }`;
 	};
 
+	const renderMailbox = () => {
+		return (
+			<ExternalLink
+				icon
+				href={ getMailboxLink() }
+				onClick={ props.onClick }
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				{ translate( 'Mailbox', { context: 'Link pointing to Gmail' } ) }
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?